candidate.py

```diff
@@ -0,0 +1,11 @@
+class cdetails:
```

Comment:
I want this to be an object

class Candidate

	def __init__(self, name, aadhar, age):
	        self._name = name
	        self._aadhar = aadhar
	        self_age = age
	        
	def is_valid(self):
	    return true